mailviews/previews.py

```diff
@@ -72,25 +76,35 @@ def register(self, cls):
 
     @property
     def urls(self):
-        urlpatterns = patterns('',
+
+        urlpatterns = [
             url(regex=r'^$',
                 view=self.list_view,
                 name='list'),
             url(regex=r'^(?P<module>.+)/(?P<preview>.+)/$',
                 view=self.detail_view,
                 name='detail'),
-        )
+        ]
+
+        if not is_django_version_greater_than_1_9:
```

Comment:
This should probably be calling the function `()`, otherwise I believe this never gets applied.